integration/spark/src/main/scala/org/apache/carbondata/spark/load/DataLoadProcessorStepOnSpark.scala

```diff
@@ -219,6 +219,7 @@ object DataLoadProcessorStepOnSpark {
       rows: Iterator[CarbonRow],
       index: Int,
       modelBroadcast: Broadcast[CarbonLoadModel],
+      isCompactionFlow: Boolean = false,
```

Comment:
why add this in between? if you keep in last then scala will take the default value if not passed, so change in DataLoadProcessBuilderOnSpark.scala will not be required

Reply:
done